src/main/java/net/dv8tion/jda/api/events/message/react/GenericMessageReactionEvent.java

```diff
@@ -121,4 +125,53 @@ public MessageReaction getReaction()
     {
         return reaction.getReactionEmote();
     }
+
+    /**
+     * Retrieves the {@link User} who added or removed the reaction.
+     * <br>If a user is known, this will return {@link #getUser()}.
+     *
+     * @return {@link RestAction} - Type: {@link User}
+     */
+    @Nonnull
+    @CheckReturnValue
+    public RestAction<User> retrieveUser()
+    {
+        if (issuer != null)
+            return new CompletedRestAction<>(getJDA(), issuer);
+        return getJDA().retrieveUserById(getUserIdLong());
+    }
+
+    /**
+     * Retrieves the {@link Member} who added or removed the reaction.
+     * <br>If a member is known, this will return {@link #getUser()}.
+     *
+     * <p>Note that banning a member will also fire {@link MessageReactionRemoveEvent} and no member will be available
+     * in those cases. An {@link net.dv8tion.jda.api.requests.ErrorResponse#UNKNOWN_MEMBER UNKNOWN_MEMBER} error response
+     * should be the failure result.
+     *
+     * @return {@link RestAction} - Type: {@link Member}
+     */
+    @Nonnull
+    @CheckReturnValue
+    public RestAction<Member> retrieveMember()
+    {
+        if (member != null)
+            return new CompletedRestAction<>(getJDA(), member);
+        return getGuild().retrieveMemberById(getUserIdLong());
+    }
+
+    /**
+     * Retrieves the message for this reaction event.
+     * <br>Simple shortcut for {@code getChannel().retrieveMessageById(getMessageId())}.
+     *
+     * <p>The {@link Message#getMember() getMember()} method will always return null for the resulting message.
```

Comment:
```suggestion
     * <p>The {@link Message#getMember() Message#getMember()} method will always return null for the resulting message.
     *  You can, instead, retrieve the member via {@link #getMember()} or {@link #retrieveMember()}.
```

Reply:
We usually do `Type.method()` in the docs and I think from context you can tell its a method on Message anyway.

Reply:
The only reason I was adding a separation here (and it should be `.`, you're right) is because in the very next line I reference a different `#getMember()`

Updated the suggestion to use a `.`